lib/utils/develop.js

```diff
@@ -46,13 +36,17 @@ module.exports = (program) => {
       test: /\.jsx?$/, // Accept either .js or .jsx files.
       exclude: babelExcludeTest,
```

Comment:
You removed `utils/babel-exclude-test.js`, so you have remove the import in this file and replace this with `exclude: /(node_modules|bower_components)/,`